modules/@apostrophecms/i18n/index.js

```diff
@@ -73,7 +89,12 @@ module.exports = {
     if (self.show) {
       self.i18next.use(apostropheI18nDebugPlugin);
     }
-    await self.i18next.init();
+
+    const i18nextOptions = self.show ? {
+      postProcess: 'apostrophei18nDebugPlugin'
```

Comment:
Do you have a second to fix the capitalization to be consistent (the I should be capitalized in all versions of the name apostropheI18nDebugPlugin)? I know it's not a new problem but we're perpetuating it.